src/tracer.js

```diff
@@ -91,6 +89,20 @@ export default class Tracer {
      * Injects the information about the given span into the carrier
      * so that the span can propogate across inter-process barriers.
      *
+     * See FORMAT_TEXT_MAP and FORMAT_BINARY for the two required carriers.
+     *
+     * Consider this pseudocode example:
+     *
+     *     var clientSpan = ...;
+     *     ...
+     *     // Inject clientSpan into a text carrier.
+     *     var textCarrier = {};
+     *     Tracer.inject(clientSpan, Tracer.FORMAT_TEXT_MAP, textCarrier);
+     *     // Incorporate the textCarrier into the outbound HTTP request header
+     *     // map.
+     *     outboundHTTPReq.headers.extend(textCarrier);
```

Comment:
Open to suggestions for the pseudocode here (variable names, etc)... but I do like the idea of providing a simple example. See also the join() example.